test/unit/test_exports/test_derived_quantities/test_derived_quantities.py

```diff
@@ -64,6 +78,46 @@ def test_all_quantities(self):
         ]
         assert header == expected_header
 
+    def test_with_units_simple(self):
```

Comment:
We might want to have a more explicit way of testing this.

Imagine that the title of each property is "coucou", this test would pass.

Maybe not here but somewhere you need to test that the title is say "Flux surface X" and not "coucou"

Reply:
Yeah I agree